### ISSUE TITLE
Hyperparameter settings during verification

### ISSUE THREAD

**Issue by cainiao123s** [ultralytics/ultralytics#22806]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

Hello, I would like to ask how the hyperparameters are set when the official model is validated on the COCO2017val dataset? I want to verify the model I trained under the same settings.

**Comment by UltralyticsAssistant** [comment]
👋 Hello @cainiao123s, thank you for your interest in Ultralytics and for your detailed question about validation settings on COCO2017val 🚀 This is an automated response to help you get started quickly — an Ultralytics engineer will also review your issue and follow up soon.

You can find many configuration and usage details in the [Docs](https://docs.ultralytics.com/), including end-to-end examples for both [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/), where many common questions about training and validation setups are already covered.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it.

Since your post is about evaluation / verification settings and hyperparameters, it falls under a custom training / usage ❓ Question. To help the team give you a precise answer, please add as much detail as possible, for example:
- Your exact command or Python code used for validation
- The model you trained and the base model it came from (e.g., YOLO11 variant)
- Any custom hyperparameters or YAML files you are using
- The metrics you are comparing against the official COCO2017val results

Also, please verify that you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/), as differences in training or validation configuration can noticeably affect COCO metrics 📊

Join the Ultralytics community where it suits you best:
- For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧  
- For in-depth discussions, check out [Discourse](https://community.ultralytics.com/)  
- Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community  

## Upgrade

Before comparing your results with official benchmarks, please upgrade to the latest `ultralytics` package (and ensure all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) are installed) in a **Python>=3.8** environment with **PyTorch>=1.8** to make sure you are aligned with the current release:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

Using one of these setups helps ensure your validation configuration is consistent with how official benchmarks are generated 🧪

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by Y-T-G** [comment]
It uses the default settings
